.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Get an env
-        uses: yetanalytics/action-setup-env@v2.0.0-SNAPSHOT
+        uses: yetanalytics/action-setup-env@v1
```

Comment:
isn't it v1.1.0 now?

Reply:
Yes, but I changed the contents of v1, as is the practice with minor version updates for GH actions.

Reply:
ah ok